### DISCUSSION TITLE
扫码后失败是怎么回事？

### DISCUSSION THREAD

**Discussion by JNCJZTT** [zhayujie/chatgpt-on-wechat#272] [Q&A]
![Uploading image.png…]()

**Comment by zhayujie** [comment]
图片没有上传成功哈

**Reply by Ten-G** [reply]
![image](https://user-images.githubusercontent.com/72859595/222372361-73d6ac5a-9280-4da3-b626-fdec9e210de7.png)

**Reply by zhayujie** [reply]
> ![image](https://user-images.githubusercontent.com/72859595/222372361-73d6ac5a-9280-4da3-b626-fdec9e210de7.png)

更新下openai

```
pip3 install --upgrade openai==0.27.0
```

**Reply by Ten-G** [reply]
![image](https://user-images.githubusercontent.com/72859595/222374087-491c21df-a0d1-4c92-bfac-988588d4e265.png)
显示没有找到0.27的版本，而且刚才是看到升级了3-5tubor我更新了下openai到2.8，然后就不能用了，重新安装了下最新代码也不可以

**Reply by zhayujie** [reply]
> ![image](https://user-images.githubusercontent.com/72859595/222374087-491c21df-a0d1-4c92-bfac-988588d4e265.png) 显示没有找到0.27的版本，而且刚才是看到升级了3-5tubor我更新了下openai到2.8，然后就不能用了，重新安装了下最新代码也不可以

你的python和pip版本是多少， 先升级下pip看看那
